testeng/resources/flow-run-simple-loadtest.groovy

```diff
@@ -0,0 +1,38 @@
+import hudson.FilePath
+import hudson.model.*
+
+def toolbox = extension."build-flow-toolbox"
+def driver_build = null
+
+/* Kick off the load test, directly copying all parameters.  Ignore the
+ * scenario where the load test is manually aborted by the user because this is
+ * considered normal behavior.
+ */
+ignore(ABORTED) {
+    driver_build = build(
+        "loadtest-driver",
+        TARGET_URL: params["TARGET_URL"],
+        TEST_COMPONENT: params["TEST_COMPONENT"],
+        REMOTE_BRANCH: params["REMOTE_BRANCH"],
+        NUM_CLIENTS: params["NUM_CLIENTS"],
+        HATCH_RATE: params["HATCH_RATE"],
+        MAX_RUN_TIME: params["MAX_RUN_TIME"],
+        LOADTEST_OVERRIDES: params["LOADTEST_OVERRIDES"]
+    )
+}
+toolbox.slurpArtifacts(driver_build)
+
+/* Read the load test logfile into memory */
+FilePath loadtest_logfile = new FilePath(
+    driver_build.workspace,
+    'edx-load-tests/results/log.txt'
+)
+String loadtest_log = loadtest_logfile.readToString()
+
+/* Kick off a build to generate a load test summary */
+def summary_build = build(
+    'loadtest-summary',
+    REMOTE_BRANCH: params["REMOTE_BRANCH"],
+    LOG: loadtest_log
```

Comment:
Skimming over the jenkins code, I can't see any obvious place where arbitrary limits are set on StringParameterValue, which TextParameterValue extends:

https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/model/StringParameterValue.java

Seems like the limit is on the size String, which is related to system memory and jvm configuration.  These logfiles grow during the lifetime of the test; I estimate a 24h test produces a 200 MB log which is still an order of magnitude smaller than the String max size.